Cargo.toml

```diff
@@ -19,6 +19,7 @@ categories = ["data-structures", "no-std"]
 
 [dependencies]
 serde = { version = "1.0", optional = true, features = ["derive"] }
+borsh = { version = ">=1.5.4, <1.6", optional = true, features = ["derive"] }
```

Comment:
We should not force an upper bound (beyond the normal `< 2` semver compatibility) because that limits future users. If we need to downgrade it in CI when testing compatible versions, we can do that, just like we do for serde already.

Speaking of which, this feature also needs to be added to `.github/workflows/ci.yml`.

Reply:
Also, we should use `default-features = false` for `no_std` compatibility.